docs/deprecated_wiki/Home.md

```diff
@@ -1,14 +0,0 @@
-> _Search this wiki_  -- Use the search box at top -- then click **Wikis** 
```

Comment:
This serves as a documentation 'landing page' for the GitHub wiki. it doesn't translate to ReadTheDocs.